### PR TITLE
Revisions based on install-to-search test

### PR DESCRIPTION
**Pull request by jon-freed**
Updated the readme.md after following and revising the instructions for setting up a local instance of the CourtListener website, populating it with data, and verifying the data can be found through the local CL search.  (Diff will not be very helpful for identifying changes due to the quantity and type.  For example, the change log section was moved to the bottom.)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,109 +1,72 @@
...
 
-#  Recommended Next Steps
-Here are a couple additional steps to get you started if you're unfamiliar with
-CourtListener or your way around a Django app.
+7. Run the following commands in the terminal to get the latest recap schema file information for solr:
+
+``` bash
+cd /opt/solr-cores/recap/conf
+sudo service solr stop
+curl https://raw.githubusercontent.com/freelawproject/courtlistener/master/Solr/conf/recap_schema.xml --Output schema.xml
```
Comment (mlissner) [inline README.md]: Hm. This fixes the problem with the recap schema, but isn't future proof if other schemas need to be updated too (there are four of them). Perhaps this is better handled just with a reference to the issue I filed earlier this morning? Something that says, "if you run into problem X, see issue Y"?

Comment (mlissner) [reply README.md]: Or maybe we just need to remove this commend and fix the template here to have the latest schema?

File: README.md [hunk 2]
```diff
@@ -148,32 +102,45 @@ You should now have some results on the landing page as well as fully searchable
 opinions!
 
 You can inspect the Solr index cores directly using your browser:
-[http://localhost:8999/solr/#/](http://localhost:8999/solr/#/)
+[http://localhost:8983/solr/#/](http://localhost:8983/solr/#/)
+
+## Modifying your local CourtListener website and contributing back to its repository
```
Comment (mlissner) [inline README.md]: How much of this section is necessary? Is it mostly just repeating what's in Github Flow? If so, I don't think we should repeat it. Best to keep our stuff as simple as possible. Maybe just include a reference instead?

File: README.md [hunk 3]
```diff
@@ -148,32 +102,45 @@ You should now have some results on the landing page as well as fully searchable
...
+Here's how to crank out a box if you've got the Requirements above. Depending on your network connection, CPU, disk, etc. this could take anywhere from 20 mins to maybe 30 mins. Be patient :-)
+
+All of the tools required to build the box using [Packer](https://packer.io) are contained in the [packer](./packer) directory of this project. [Ansible](https://github.com/ansible/ansible) is installed into the VM image itself, so there's no need to have it installed on your local machine.
+
+  1. Grab the latest Free Law Machine source:
 
     `git clone https://github.com/freelawproject/freelawmachine`
 
-  1. Jump into the packer directory:
+  2. Jump into the packer directory:
```
Comment (mlissner) [inline README.md]: FYI, in markdown, the numbers don't actually matter. I find it easier just to number everything 1., 1., .1, etc. Then you don't have to tweak things like this.

File: README.md (hunk 2 above)
Comment (jon-freed) [reply README.md]: > How much is this section is necessary?

None of it, but is it more helpful than not to people who could make valuable contributions? 

> Is it mostly just repeating what's in Github Flow?

The concepts are in the GitHub flow, but these specifics aren't there.

File: README.md (hunk 3 above)
Comment (jon-freed) [reply README.md]: I have found that the 1.,1.,1. pattern does not always work, like when there are intervening newlines or code sections.

Comment (voutilad) [reply README.md]: Numbering everything with `1.` goofs up plain-text readability. My opinion is to err on the side of explicit numbering.

Comment (johnhawkinson) [reply README.md]: > I have found that the 1.,1.,1. pattern does not always work, like when there are intervening newlines or code sections.

In that case, you are in even more trouble. The potential "2." following a newline will be then read by Markdown as  starting to number a new section and will render as `1.`. That is, "does not always work" is not a reason to use an explicit number like "2."

I don't have a strong opinion on the larger question though :)

But if this is all we're talking about, well, it probably means we're bikeshedding.

**Comment by jon-freed** [comment]
So...   @mlissner et al., have all concerns been addressed?  If no, could the concerned person please clarify what remains outstanding?

If yes, then I prefer that somebody else do the squash and merge action.  Although I now have the GitHub permission needed to do that here, I would rather not do it for changes that I personally proposed in a repository I do not own.

**Comment by mlissner** [comment]
This is a huge improvement. Thank you again for all of this!

**Pull request merged**